about.html

```diff
@@ -0,0 +1,170 @@
+                <h1 class="pt-5">"Hello! Sharing World"</h1>
+                <h1 class="display-1 text-center">Aanchal Global<br> Impex Expo 2021</h1>
+                <div class="container p-3 bg-white d-flex justify-content-center border-2">
+                    <p class="text-center d-none d-lg-block">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Dolores, autem fuga explicabo nobis
+                        dolorum ipsam vitae veniam blanditiis, eum quam et iusto porro, ut corrupti voluptate non!
+                        Laboriosam repellat aut sint ab debitis earum dignissimos aperiam similique est unde, explicabo
+                        sapiente accusantium nulla quis sunt labore impedit cum sit. Ipsam recusandae eaque fuga nostrum
+                        repellendus ab cumque eum iusto, ea tempore veritatis enim voluptates, aliquam eius animi?
+                        Officia expedita laborum reiciendis quia, nemo porro temporibus earum provident autem
+                        dignissimos modi iste cupiditate ratione optio eius incidunt non doloribus aliquam! Deleniti vel
+                        magnam consequuntur iusto quam magni voluptatem rem aspernatur reiciendis! Consequuntur quisquam
+                        labore vel voluptatem molestiae itaque, dicta vero, quibusdam cupiditate aliquam alias eos quos
+                        magni quae excepturi? Dolorem veritatis nisi voluptates suscipit recusandae aperiam pariatur
+                        eaque officiis. Odio ad ea iusto! Magni iusto praesentium deleniti ea in tenetur dolorem aperiam
+                        provident dicta enim? Libero iste atque eveniet enim sequi.</p>
+                    <p class="text-center d-lg-none d-sm-block ">Lorem ipsum dolor sit amet consectetur adipisicing
+                        elit. Dolores, autem fuga explicabo nobis
+                        dolorum ipsam vitae veniam blanditiis, eum quam et iusto porro, ut corrupti voluptate non!
+                        Laboriosam repellat aut sint ab debitis earum dignissimos aperiam similique est unde, explicabo
+                        sapiente accusantium nulla quis sunt labore impedit cum sit. Ipsam recusandae eaque fuga nostrum
+                        repellendus ab cumque eum iusto, ea tempore veritatis enim voluptates, aliquam eius animi?
+                        Officia expedita laborum reiciendis quia, nemo porro temporibus earum </p>
+                </div>
+                <p class="text-center pt-3">Please contact us per Email for any further queries</p>
+                <strong class="text-center pb-3">hello@aanchalenterprises.com</strong>
+            </div>
+
+        </section>
+        <section class="container-fluid bg-white d-flex flex-column justify-content-center">
+            <h2 class="mt-5 about-heading text-center">Aanchal Enterprises Global Export Summit 2021</h2>
```

Comment:
- [ ] For all the headings on this page I would like you to consider incrementing the font-weight as can observe from the design given